Source/Core/DolphinQt2/GameList/GameFile.h

```diff
@@ -65,18 +67,95 @@ class GameFile final
   QString GetLongMaker() const { return GetBannerString(m_long_makers); }
   QString GetDescription() const { return GetBannerString(m_descriptions); }
   QString GetShortName(DiscIO::Language lang) const { return m_short_names[lang]; }
+  QMap<DiscIO::Language, QString> GetShortNames() { return m_short_names; }
   QString GetShortMaker(DiscIO::Language lang) const { return m_short_makers[lang]; }
+  QMap<DiscIO::Language, QString> GetShortMakers() { return m_short_makers; }
   QString GetLongName(DiscIO::Language lang) const { return m_long_names[lang]; }
+  QMap<DiscIO::Language, QString> GetLongNames() { return m_long_names; }
   QString GetLongMaker(DiscIO::Language lang) const { return m_long_makers[lang]; }
+  QMap<DiscIO::Language, QString> GetLongMakers() { return m_long_makers; }
   QString GetDescription(DiscIO::Language lang) const { return m_descriptions[lang]; }
+  QMap<DiscIO::Language, QString> GetDescriptions() { return m_descriptions; }
   bool Install();
   bool Uninstall();
   bool ExportWiiSave();
 
+  class Cache
```

Comment:
I'm not sure this particularly belongs directly inside `GameFile`, especially since it's supposed to represent a singular entity.